.node_version

```diff
@@ -1 +1 @@
-v16.20.2
+v18.0.0
```

Comment:
Not sure which value we actually want here! Latest 18.x?

Reply:
We might as well match what we use in build container: `18.20.6`